### ISSUE TITLE
Missing @param type results in incorrect reporting of param name

### ISSUE THREAD

**Issue by nzakas** [eslint/doctrine#22]
If I have a JSDoc comment like this:

``` js
/**
 * My desc
 * @param foo Something descriptive
 */
```

The resulting node is:

``` js
{
    "description": "My description",
    "tags": [
        {
            "title": "param",
            "description": "descriptive",
            "type": {
                "type": "NameExpression",
                "name": "foo"
            },
            "name": "Something"
        }
    ]
}
```

I would have expected it to say that there is no type and that the name is "foo" rather than "something".

According to the JSDoc documentation (http://usejsdoc.org/tags-param.html), when there are no braces, the first identifier should be the parameter name and the rest should be the description.
